### DISCUSSION TITLE
Do you have any puzzles and clvm tools for C++ and JavaScript?

### DISCUSSION THREAD

**Discussion by Jimmy01240397** [Chia-Network/chia-blockchain#8664] [Support]
Do you have any puzzles and clvm tools for C++ and JavaScript?
![image](https://user-images.githubusercontent.com/57281249/135596348-43f817fc-a292-4e5b-a2d6-668a28541f94.png)

**Comment by BrandtH22** [comment]
@Jimmy01240397 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
